admin/init.sls

```diff
@@ -22,22 +22,37 @@ UTC:
     - mode: 644
     - source: salt://{{ tpldir }}/files/hosts
 
-sshkeys-dir:
-  file.directory:
-    - name: {{ root.home }}/.ssh
-    - user: {{ root.user }}
-    - group: {{ root.group }}
-    - mode: 700
-
-sshkeys:
+/etc/sshd_config:
   file.managed:
-    - name: {{ root.home }}/.ssh/authorized_keys
-    - user: {{ root.user }}
-    - group: {{ root.group }}
-    - mode: 600
-    - contents:
-      {% for ssh_user in admin.ssh_users %}
-      - {% include tpldir ~ '/files/ssh/' ~ ssh_user ~ '.pub' %}
-      {% endfor %}
-    - require:
-      - file: sshkeys-dir
+    - user: root
+    {% if grains['os'] == 'MacOS' %}
+    - group: wheel
+    {% elif grains['os'] == 'Ubuntu' %}
+    - group: root
+    {% endif %}
+    - mode: 644
+    - source: salt://{{ tpldir }}/files/sshd_config
+
+wheel:
```

Comment:
I agree with your comment about [only providing some users root access](https://github.com/servo/saltfs/pull/628#discussion_r147214123). To prevent confusion, let's rename this group to `sshusers`, and reserve the `wheel` group for users that will have root perms.